pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotControllerHealthCheck.java

```diff
@@ -119,6 +121,20 @@ public long getUptime() {
     return uptime.getSeconds();
   }
 
+  @GET
+  @Path("/pinot-controller/invalid/configs")
```

Comment:
This is an API on controller, so we can simplify it to
```suggestion
  @Path("invalidconfigs")
```